vendor/assets/stylesheets/ombulabs/styleguide.scss

```diff
@@ -4,7 +4,7 @@
 ******************************/
 
 // Google Web Fonts
-@import 'https://fonts.googleapis.com/css?family=Lato:700|Open+Sans:300,300i,400,400i,700,700i&display=swap';
+@import url("https://fonts.googleapis.com/css?family=Lato:700|Open+Sans:300,300i,400,400i,700,700i&display=swap");
```

Comment:
It looks like you don't need the quotes here when using the url function https://www.developerdrive.com/how-to-easily-use-google-fonts-with-sass/